.github/workflows/release.yaml

```diff
@@ -0,0 +1,93 @@
+name: Release Flow
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  publish-packages:
+    name: Push Packages
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python-version: ["3.10"]
+    steps:
+      - uses: actions/checkout@master
+      - uses: actions/setup-python@v3
+        with:
+          python-version: ${{ matrix.python-versions }}
+      - uses: addnab/docker-run-action@v3
+        with:
+            image: valory/open-autonomy-user:latest
+            options: -v ${{ github.workspace }}:/work
+            run: |
+              echo "Pushing Packages"
+              cd /work
+              export AUTHOR=$(grep 'service' packages/packages.json | awk -F/ '{print $2}' | head -1)
+              autonomy init --reset --author $AUTHOR --ipfs --remote
+              autonomy push-all
+  publish-images:
+    name: Publish Docker Images
+    runs-on: ${{ matrix.os }}
+    needs:
+      - "publish-packages"
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python-version: ["3.10"]
+    env:
+        VERSION:  ${{github.event.release.tag_name}}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up tag and vars
+        uses: addnab/docker-run-action@v3
+        with:
+            image: valory/open-autonomy-user:latest
+            options: -v ${{ github.workspace }}:/work
+            run: |
+              echo "Setting Tag Images"
+              cd /work
+              apt-get update && apt-get install git -y || exit 1
+              export TAG=$(git describe --exact-match --tags $(git rev-parse HEAD)) || exit 1
+              if [ $? -eq 0 ]; then
+                  export TAG=`echo $TAG | sed 's/^v//'`
+              else
+                  echo "You are not on a tagged branch"
+                  exit 1
```

Comment:
Not the prettiest but;
![image](https://user-images.githubusercontent.com/35799987/214417647-94522122-372c-4b84-ae63-4c327df45a48.png)